src/main/java/com/streisky/virtualshowcasebackend/entity/product/Product.java

```diff
@@ -62,6 +63,37 @@ private List<ImageDTO> mapImagesDTO() {
         return getImages() != null ? getImages().stream().map(Image::toDTO).toList() : null;
     }
 
+    public void updateData(ProductDTO productDTO) {
+        if (Objects.nonNull(productDTO.description())) {
+            setDescription(productDTO.description());
+        }
+
+        if (Objects.nonNull(productDTO.amount())) {
+            setAmount(productDTO.amount());
+        }
+
+        if (Objects.nonNull(productDTO.images())) {
```

Comment:
A (totally optional) suggestion would be to separate this block into another function to make it easier to read. Something like `updateImagesData`.

Reply:
Changed as suggested.